grails-app/views/occurrence/list.gsp

```diff
@@ -276,7 +276,7 @@
                                     <g:set var="spatialType" value="${params.wkt =~ /^\w+/}"/>
 
                                     <a href="${alatag.getQueryStringForWktRemove()}">
-                                        <button class="erk-button erk-button--light btn-mini tooltips" title="Click to remove this filter">
+                                        <button class="erk-button erk-button--light btn-mini tooltips">
```

Comment:
I haven't played with these myself but if I rember it correctly, tooltips class is for displaying title property value in a tooltip, so not used here any more.